src/app/information/information.component.ts

```diff
@@ -0,0 +1,66 @@
+import { Component, OnInit } from '@angular/core';
+import { FormGroup, Validators, FormBuilder } from '@angular/forms';
+import { Router } from '@angular/router';
+import { UserServiceService } from '../services/user-service.service';
+@Component({
+  selector: 'app-information',
+  templateUrl: './information.component.html',
+  styleUrls: ['./information.component.sass']
+})
+export class InformationComponent implements OnInit {
+  form: FormGroup;
+  togglePage = {
```

Comment:
wtf logic with this var and method related to it

create `state` variable
types for this state named like this `<'name' | 'email' | 'phone' | 'image'>`

in method set one of this states and show it used ngIf or ngSwitch directive